.gitignore

```diff
@@ -73,3 +73,5 @@ target/
 .DS_Store
 src/.DS_Store
 
+.out
```

Comment:
The folder `out` should be ignored and removed from the current commit

Reply:
removed directory